src/packaging/post-install.sh

```diff
@@ -0,0 +1,4 @@
+set -e
+
+/bin/systemctl daemon-reload
+/bin/systemctl start knoxAutoPolicy.service
```

Comment:
Also if possible/required, add below line 

/bin/systemctl enable knoxAutoPolicy.service

Reply:
Enabling the service will automatically start the discovery-engine on each system startup, I think that might not be required by the user. Wdyt?

Reply:
Yeah, lets not start by default.